### PR TITLE
Refactoring Configs into Security Settings for COOP and COEP

### PR DESCRIPTION
**Pull request by eozmen410**

### CODE REVIEW

File: wicket-core/src/main/java/org/apache/wicket/Application.java [hunk 1]
```diff
@@ -763,6 +767,8 @@ public final void initApplication()
 		applicationListeners.onAfterInitialized(this);
 
 		validateInit();
+
+		securityInit();
```
Comment (gchatz22) [inline wicket-core/src/main/java/org/apache/wicket/Application.java]: nit: I see from above the `init` methods don't have whitespace in between but it is not consistent either

Comment (gchatz22) [inline wicket-core/src/main/java/org/apache/wicket/Application.java]: nit: I see from the above `init` methods they do not have white space in between, but to your defense it is not consistent

File: wicket-core/src/main/java/org/apache/wicket/Application.java [hunk 2]
```diff
@@ -787,6 +793,28 @@ protected void validateInit()
...
+	 */
+	protected void securityInit()
+	{
+		// enable coop and coep listeners if specified in security settings
+		CrossOriginOpenerPolicyConfiguration coopConfig = getSecuritySettings()
+			.getCrossOriginOpenerPolicyConfiguration();
+		if (coopConfig.isEnabled())
+		{
+			getRequestCycleListeners().add(new CoopRequestCycleListener(coopConfig));
+		}
```
Comment (gchatz22) [inline wicket-core/src/main/java/org/apache/wicket/Application.java:808]: nit: new line to separate COOP and COEP ?

File: wicket-core/src/main/java/org/apache/wicket/settings/SecuritySettings.java [hunk 3]
```diff
@@ -57,7 +62,7 @@
 
 	/** factory for creating crypt objects */
 	private ICryptFactory cryptFactory;
-
```
Comment (gchatz22) [inline wicket-core/src/main/java/org/apache/wicket/settings/SecuritySettings.java:60]: nit: do you need extra white space?

File: wicket-core/src/main/java/org/apache/wicket/settings/SecuritySettings.java [hunk 4]
```diff
@@ -24,12 +24,17 @@
...
+import org.apache.wicket.coep.CrossOriginEmbedderPolicyConfiguration.CoepMode;
+import org.apache.wicket.coop.CrossOriginOpenerPolicyConfiguration;
+import org.apache.wicket.coop.CrossOriginOpenerPolicyConfiguration.CoopMode;
 import org.apache.wicket.core.random.DefaultSecureRandomSupplier;
 import org.apache.wicket.core.random.ISecureRandomSupplier;
 import org.apache.wicket.core.util.crypt.KeyInSessionSunJceCryptFactory;
 import org.apache.wicket.util.crypt.ICryptFactory;
 import org.apache.wicket.util.lang.Args;
 
+
```
Comment (gchatz22) [inline wicket-core/src/main/java/org/apache/wicket/settings/SecuritySettings.java]: nit: do you need extra white space?

**Review by gchatz22 — COMMENTED** [review]
Great work, some very nit style fixes!!

**Review by salcho — APPROVED** [review]
Nice! Looking good, let's address some of the extra whitespace that Giannis has pointed out and update the PR 👍

**Pull request merged**